src/screens/Crypto/Crypto.tsx

```diff
@@ -52,7 +52,7 @@ const Crypto = ({ navigation }: Props) => {
   return (
     <SafeAreaView>
       <Background>
-        <ArrowButton onPress={navigation.goBack.bind(null)} >
+        <ArrowButton onPress={() => navigation.goBack()}>
```

Comment:
[Question] navigation.goBack, doesn't it get executed just by putting it as onPress={navigation.goBack} since it's a callback?